examples/tests/valid/dynamo_awscdk.w

```diff
@@ -0,0 +1,96 @@
+/*\
+skip: true
+\*/
+// this example only works on AWS (intentionally)
+
+bring "aws-cdk-lib" as awscdk;
+bring aws;
+bring cloud;
+bring util;
+
+enum AttributeType {
+  String,
+  Number,
+  Binary,
+}
+
+struct Attribute {
+  type: AttributeType;
+  value: Json;
+}
+
+class DynamoTable {
+  table: awscdk.aws_dynamodb.Table;
+  tableName: str;
+  init() {
+    let target = util.env("WING_TARGET");
+    if target != "awscdk" {
+      throw("Unsupported target: ${target} (expected 'awscdk')");
+    }
+
+    this.table = new awscdk.aws_dynamodb.Table(
+      tableName: this.node.addr,
+      billingMode: awscdk.aws_dynamodb.BillingMode.PAY_PER_REQUEST,
+      partitionKey: awscdk.aws_dynamodb.Attribute {
+        name: "Flavor",
+        type: awscdk.aws_dynamodb.AttributeType.STRING,
+      },
+    );
+    this.tableName = this.table.tableName;
+  }
+
+  _bind(host: std.IInflightHost, ops: Array<str>) {
```

Comment:
@Chriscbr it feels weird that people need to override an internal method... maybe we should formalize it? (At least change it to `bind()`.

(Would be nice to have something like a "private protected" method which can only be overridden by derived classes but not called from outside)

Reply:
I agree. Let's make it public

Reply:
Do you want me to make the changes in this PR, or should I open another one?

Reply:
In another PR is fine 👍

Reply:
Open a issue #3775 and a PR #3776